src/functions/utils/assembleSupSub.js

```diff
@@ -17,8 +17,6 @@ export const assembleSupSub = (
     slant: number,
     baseShift: number,
 ): DomSpan => {
-    // IE 8 clips \int if it is in a display: inline-block. We wrap it
-    // in a new span so it is an inline, and works.
```

Comment:
Do we still need to wrap it?

Reply:
@kevinbarabash  One commit early in this PR removed that wrap. The result was that the `subGroup` below an `\int` or a `\sum` had their top padding removed and were squashed up against the big operator.  That happened in both Chrome and Firefox. 

@ylemkimon then put the wrap span back in and removed the comment stating that it was there only for IE. I think that was the right move.